tool/planet/constants.go

```diff
@@ -434,7 +434,7 @@ const (
 	DefaultVxlanPort = 8472
 
 	// DefaultFeatureGates is the default set of component feature gates
-	DefaultFeatureGates = "AllAlpha=true,APIResponseCompression=false,BoundServiceAccountTokenVolume=false,CSIMigration=false,KubeletPodResources=false,IPv6DualStack=false,RemoveSelfLink=false"
+	DefaultFeatureGates = "AllAlpha=true,APIResponseCompression=false,BoundServiceAccountTokenVolume=false,CSIMigration=false,KubeletPodResources=false,IPv6DualStack=false,RemoveSelfLink=false,StorageVersionAPI=false"
```

Comment:
Since you are touching these it might be a good time to clean these up unless there is a desire to keep them intentionally off still :) 

https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

- `BoundServiceAccountTokenVolume` defaults to `true` in 1.21
- `CSIMigration` defaults to `true` as of 1.17
- `KubeletPodResources` has always been `true` actually, even since it was added as alpha in 1.13 (it's beta now) so perhaps a reason it was off and @knisbet or @r0mant remember why.
- `IPv6DualStack` defaults to `true` in 1.21 and went beta
- `RemoveSelfLink` defaults to`true` as of 1.20

Reply:
Thanks for pointing these out. I'll take a look and see what's safe to remove.